torch/autograd/gradcheck.py

```diff
@@ -401,6 +445,27 @@ def not_reentrant_error(error_str=''):
         if out_is_complex and not reentrant_with_imag_grad_out:
             return not_reentrant_error(' (calculated using complex valued grad output)')
 
+        if check_forward:
+            try:
+                fw_analytical = get_analytical_jacobian_fw(fn, tupled_inputs, o)
+            except RuntimeError as e:
+                msg = str(e)
+                if "Trying to use forward prop with" in msg:
+                    warnings.warn("Failed to compute gradcheck using fw mode: {}".format(msg), stacklevel=stacklevel)
```

Comment:
Yes.
Because warnings with default setting show only one line (and not the full stack), I think that pointing to the place where gradcheck is called is more important than this internal line within the gradcheck function.